includes/reports/overview/list-table/class-sensei-reports-overview-list-table-factory.php

```diff
@@ -26,6 +26,7 @@ class Sensei_Reports_Overview_List_Table_Factory {
 	 */
 	public function create( string $type ) {
 		switch ( $type ) {
+			case 'users':
```

Comment:
I don't think we should account for technical debt generated from outside the factory. Could this be fixed in `Sensei_Analysis`?